rmgpy/chemkin.pyx

```diff
@@ -449,6 +452,25 @@ def _read_kinetics_line(line, reaction, species_dict, Eunits, kunits, klow_units
         # Duplicate reaction
         reaction.duplicate = True
 
+    elif 'COV' in line:
+        k = kinetics['arrhenius high']
```

Comment:
If you had a `STICK` line before this, then the `'arrhenius high'` key will have been deleted from `kinetics` (line 583) and so this probably won't work on files like https://www.detchem.com/public/files/mechanisms/11_CH4_O2_ReducedGas_Quiceno2006/sm_CH4_O2_Ptwire_2006_chemkin.txt which have 
```
REACTIONS  KJOULES/MOLE  MWOFF 
H2 + _Pt_ + _Pt_ => H_Pt + H_Pt             4.600E-02  0.000      0.000 
    STICK 
    COV / _Pt_                               0.000E+00  -1.000     -0.000 / 
```

Reply:
yes, I discovered this and hoped no one would notice

Reply:
Now fixed.